playground/db/README.md

```diff
@@ -0,0 +1,13 @@
+# db
```

Comment:
Should we move this into the playground readme or link to it? I don't think I'll be able to find it otherwise.